src/jit/block.h

```diff
@@ -438,6 +438,8 @@ struct BasicBlock : private LIR::Range
 #define BBF_CLONED_FINALLY_BEGIN 0x100000000 // First block of a cloned finally region
 #define BBF_CLONED_FINALLY_END 0x200000000   // Last block of a cloned finally region
 
+#define BBF_DOMINATED_BY_NOT_NORMAL_ENTRY 0x400000000 // Full set of dominators contains not normal entry.
+
```

Comment:
>  it is a bit unclear because we include call_finally block for ARM and I don't think that we could call it exceptional.

Well, you could say "Special" and define that somewhere :-)

> Do we expect better performance if we SetVolatileHint on local variables that live in finally blocks?

Not sure. Is it perf, or correctness?